src/app/[locale]/_utils/common.ts

```diff
@@ -22,3 +23,21 @@ export const getSessionStatus = (sessionData: SessionState | null): 'on' | 'off'
 
 export const getAccessStatus = (sessionData: SessionState | null): 'unlocked' | 'locked' =>
   sessionData?.access_enabled ? 'unlocked' : 'locked';
+
+export const getLoginFlow = (loginInfo: LoginInfo): string => {
+  if (loginInfo) {
+    storageLoginInfoOps.delete();
```

Comment:
Why is thi inside this function? 🤔 Could it be misleading in the long run? Bringing to edge cases or critical path?